components/homme/cmake/machineFiles/spock-gpumpi.cmake

```diff
@@ -0,0 +1,62 @@
+#interactive job
+#bsub -W 2:00 -nnodes 1 -P cli115 -Is /bin/bash
+
+
+#cmake -C ~/acme-fork-lb/components/homme/cmake/machineFiles/summit.cmake -DHOMMEXX_MPI_ON_DEVICE=FALSE ~/acme-fork-lb/components/homme/
+
+#SET (HOMMEXX_MPI_ON_DEVICE FALSE CACHE BOOL "")
+SET (HOMMEXX_CUDA_MAX_WARP_PER_TEAM "16" CACHE STRING  "")
+
+SET (NETCDF_DIR $ENV{OLCF_NETCDF_FORTRAN_ROOT} CACHE FILEPATH "")
+SET (HDF5_DIR $ENV{OLCF_HDF5_ROOT} CACHE FILEPATH "")
+
+SET(BUILD_HOMME_WITHOUT_PIOLIBRARY TRUE CACHE BOOL "")
+
+SET(HOMME_FIND_BLASLAPACK TRUE CACHE BOOL "")
+
+SET(WITH_PNETCDF FALSE CACHE FILEPATH "")
+
+SET(USE_QUEUING FALSE CACHE BOOL "")
+
+SET(BUILD_HOMME_PREQX_KOKKOS TRUE CACHE BOOL "")
+SET(BUILD_HOMME_THETA_KOKKOS TRUE CACHE BOOL "")
+SET(HOMME_ENABLE_COMPOSE FALSE CACHE BOOL "")
+
+#SET (HOMMEXX_BFB_TESTING TRUE CACHE BOOL "")
+
+SET(USE_TRILINOS OFF CACHE BOOL "")
+
+SET(HIP TRUE CACHE BOOL "")
+#SET(Kokkos_ENABLE_OPENMP OFF CACHE BOOL "")
+#SET(Kokkos_ENABLE_CUDA OFF CACHE BOOL "")
+#SET(Kokkos_ENABLE_CUDA_LAMBDA OFF CACHE BOOL "")
+#SET(Kokkos_ARCH_VEGA908 ON CACHE BOOL "")
+#SET(Kokkos_ENABLE_EXPLICIT_INSTANTIATION OFF CACHE BOOL "")
+
+SET(CMAKE_C_COMPILER "cc" CACHE STRING "")
+SET(CMAKE_Fortran_COMPILER "ftn" CACHE STRING "")
+SET(CMAKE_CXX_COMPILER "hipcc" CACHE STRING "")
+#SET(CMAKE_CXX_COMPILER "/ccs/home/onguba/kokkos/bin/nvcc_wrapper" CACHE STRING "")
+SET(E3SM_KOKKOS_PATH "/ccs/home/onguba/kokkos-example-spock-hipcc2/bld-hipcc" CACHE STRING "")
```

Comment:
I had removed dependence on `onguba`-based things in the crusher file. spock is I think irrelevant at this point, so I don't mind this line. But we could also consider not including the spock file in this PR. Up to you, though.